src/common/imageio_rawspeed.cc

```diff
@@ -267,7 +283,7 @@ dt_imageio_retval_t dt_imageio_open_rawspeed_sraw(dt_image_t *img, RawImage r, d
            * we need to copy data from only channel to each of 3 channels
            */
 
-          out[k] = MAX(0.0f, (((float)(*in)) - black) / scale);
+          out[k] = (float)*in / (float)UINT16_MAX;
```

Comment:
wait, how does the pipeline work? you don't pass on the unchanged uint16_t value, but you rescale it to 0..1? is that required for something? i'd rather use the exact value and pass on a white point, too.

noise profiles will depend on the absolute scale, also filling in random bits to prevent banding will need to know about the unscaled input.